app/service_utils.py

```diff
@@ -53,7 +53,7 @@ def validate_service(service):
 
     data = drop_foreign_fields(
         data,
-        ['service_id', 'supplierName', 'links', 'frameworkName', 'updatedAt'])
+        ['service_id', 'supplierName', 'links', 'frameworkSlug', 'frameworkName', 'updatedAt'])
```

Comment:
`updatedAt` and `createdAt` are both in the schema and are removed from stored service data by the `.update_from_json`, so it's more likely that there's no need to drop `updatedAt` here. But it's hard to trace exactly what's going on, instead I've changed the tests to check that `updatedAt` and `createdAt` are not saved to the JSON data and fixed the code so the tests pass. I think we need to rethink how we deal with fields later.